testgrid/config-upload.sh

```diff
@@ -18,5 +18,6 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
+bazel clean --expunge
 bazel build //testgrid/config:testgrid-config
 gsutil cp bazel-genfiles/testgrid/config/testgrid-config gs://k8s-testgrid/config
```

Comment:
Let's create an `upload target` and replace this script with a `bazel run //testgrid/config:upload` command using the bazel scenario